server/migrations/20200410175455-create-role.js

```diff
@@ -0,0 +1,30 @@
+export function up(queryInterface, Sequelize) {
+  return queryInterface.createTable('roles', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER,
+    },
+    name: {
+      allowNull: false,
+      type: Sequelize.STRING,
+      unique: true,
+    },
+    permissions: {
```

Comment:
Yes, a Role can have multiple permissions. For now, they are all saved as a single string, separated by a comma... for example: `create-trip,approve-trip,delete-trip`

I am trying to keep it simple for now, since I will add the permissions functionality in another PR.